api/config.go

```diff
@@ -56,9 +71,13 @@ Failure Threshold: {{.FailureThreshold}}
 Exec Probe: {{or .Exec "N/A"}}
 HTTP GET Probe: {{or .HTTPGet "N/A"}}
 TCP Socket Probe: {{or .TCPSocket "N/A"}}`)
-	if err != nil { panic(err) }
+	if err != nil {
+		panic(err)
+	}
 	err = tmpl.Execute(&doc, h)
```

Comment:
Super nit-pick, but we seem to prefer the one-liner version of this Go construct when possible:
```go
if err = tmpl.Execute(&doc, h); err != nil {
        panic(err)
}
```